pkgs/applications/misc/protonup-qt/default.nix

```diff
@@ -0,0 +1,33 @@
+{ appimageTools, fetchurl, lib }:
+let
+  pname = "protonup-qt";
+  version = "2.7.4";
+  src = fetchurl {
+    url = "https://github.com/DavidoTek/ProtonUp-Qt/releases/download/v${version}/ProtonUp-Qt-${version}-x86_64.AppImage";
+    sha256 = "yKc+KOQfqciqULnChVLf6y9npoSYM6Fmu7mYGEKmpkA=";
+  };
+  appimageContents = appimageTools.extractType2 { inherit pname version src; };
+in
+appimageTools.wrapType2 rec {
+  inherit pname version src;
+
+  extraInstallCommands = ''
+    mv $out/bin/{${pname}-${version},${pname}}
+    mkdir -p $out/share/{applications,pixmaps}
+    cp ${appimageContents}/net.davidotek.pupgui2.desktop $out/share/applications/${pname}.desktop
+    cp ${appimageContents}/net.davidotek.pupgui2.png $out/share/pixmaps/${pname}.png
+    substituteInPlace $out/share/applications/${pname}.desktop \
+      --replace 'Exec=net.davidotek.pupgui2' 'Exec=${pname}' \
+      --replace 'Icon=net.davidotek.pupgui2' 'Icon=${pname}'
+  '';
+
+  meta = with lib; {
+    homepage = "https://davidotek.github.io/protonup-qt/";
+    description = "Install and manage Proton-GE and Luxtorpeda for Steam and Wine-GE for Lutris with this graphical user interface.";
+    license = licenses.gpl3;
+    sourceProvenance = with sourceTypes; [ binaryNativeCode ];
+    mainProgram = "protonup-qt";
+    platforms = [ "x86_64-linux" ];
+    maintainers = with maintainers; [ quasigod-io ];
```

Comment:
This has broken CI in the past hour. Fixed in f427d0b71b6f788769320391cb779f6387d1ecd9c. Please re-add yourself to the maintainers list in `maintainers/maintainer-list.nix` and in the package, in a new PR.

Reply:
I had updated my name in the maintainer list since I created this pr and never updated, mb. Creating new PR now.

Reply:
Opened PR #216838